pkg/setting/setting.go

```diff
@@ -419,14 +428,18 @@ type Cfg struct {
 	GeomapEnableCustomBaseLayers bool
 
 	// Unified Alerting
-	AdminConfigPollInterval        time.Duration
-	AlertmanagerConfigPollInterval time.Duration
-	HAListenAddr                   string
-	HAAdvertiseAddr                string
-	HAPeers                        []string
-	HAPeerTimeout                  time.Duration
-	HAGossipInterval               time.Duration
-	HAPushPullInterval             time.Duration
+	AdminConfigPollInterval          time.Duration
+	AlertmanagerConfigPollInterval   time.Duration
+	HAListenAddr                     string
+	HAAdvertiseAddr                  string
+	HAPeers                          []string
+	HAPeerTimeout                    time.Duration
+	HAGossipInterval                 time.Duration
+	HAPushPullInterval               time.Duration
+	UnifiedAlertingMaxAttempts       int64
```

Comment:
Personally, I prefer be specific about it so that people don't get confused when browsing the code. I have noticed that people usually confuse the two systems eg when they report issues they don't specify which alerting system use. I feel that also other developers outside the alerting team also do.
In addition to this, given that we want to [move all settings to from package level vars in settings package to the setting.Cfg](https://github.com/grafana/grafana/tree/main/pkg#settings-refactoring), we have to move those of the legacy alerting as well.
However, I don't mind changing it, if we decide as a team that we prefer it.

Reply:
I have opened #39350 so that we can move to their own struct - WDYT?